test/integration/spec/rtcpeerconnection.js

```diff
@@ -269,12 +273,10 @@ describe(description, function() {
       const answer1 = await pc.createAnswer();
       await pc.setLocalDescription(answer1);
       const tracksBefore = flatMap(pc.getRemoteStreams(), stream => stream.getTracks());
-
       await pc.setRemoteDescription(offer2);
       const answer2 = await pc.createAnswer();
       await pc.setLocalDescription(answer2);
       const tracksAfter = flatMap(pc.getRemoteStreams(), stream => stream.getTracks());
-
```

Comment:
Can you please reintroduce the two deleted lines? They help organize this section.